### ISSUE TITLE
Git version problem with Ubuntu 18.04

### ISSUE THREAD

**Issue by sercantor** [CloudSnorkel/cdk-github-runners#50]
## Problem
Because the images use `ubuntu:18.04`, the git version is `2.17.1`, `checkout@v3` throws this warning message:

```
The repository will be downloaded using the GitHub REST API
To create a local Git repository instead, add Git 2.18 or higher to the PATH
```

This causes some problems with committing inside the `self-hosted` runner, meaning you will get an error like this when you try to commit:

```
fatal: not a git repository (or any of the parent directories): .git
```

I don't know if there's a specific reason we use 18.04 but is it possible to bump the ubuntu version to 20.04 since it is LTS as well?

**Comment by kichik** [comment]
Should be easy enough to update to latest with:

```
add-apt-repository ppa:git-core/ppa
apt update
apt install git
```

I'll add it later today unless you want to create a PR before that.

With the latest build image customization code, you can also override the base image for a quicker solution.

**Comment by sercantor** [comment]
Yeah, that is a workaround that I tried as well, though you need to run 
```
apt install software-properties-common
```
or you'll get 
```
add-apt-repository: command not found
```
The build customization is really great, but I love that this library has very sane defaults, so I'll open a PR just changing the versions to 20.04

**Comment by sercantor** [comment]
Oh, I didn't know that, if that's the case, you're right.  I'll close this as the default image now makes sense to me, thank you 🙂👍

**Comment by kichik** [comment]
Turns out `ubuntu-latest` is actually `ubuntu-20.04`, specifying just `linux` hangs, and all Ubuntu versions contain the latest git 2.36.1. I'll open a PR to update to latest git on all images.
